styles.css

```diff
@@ -532,3 +531,113 @@ textarea {
   justify-content: end;
   padding: 25px 0;
 }
+
+/* Project Details */
+
+.projectDetailContainer {
+  background-color: rgba(0, 0, 0, 0.4);
+  width: 100vw;
+  height: 100vh;
+  position: fixed;
+}
+
+.projectDetail {
+  background-color: #fff;
+  width: 90%;
+  height: 90vh;
+  position: fixed;
+  transform: translate(-50%, -50%);
+  z-index: 2;
+  border-radius: 2rem;
+  top: 50%;
+  left: 50%;
+  padding: 1.7rem;
+  overflow-y: scroll;
+}
```

Comment:
- [ ] great job with the desktop popup, however please notice that in the figma design the topic, paragraph and buttons are aligned with the image, currently in your design the only thing is that they are not aligned properly pleae see :point_down: 
![screenshot](https://user-images.githubusercontent.com/97521557/216132050-a3f7bfde-c8d9-41a1-95ec-573e6132add5.png)

how it should be :point_down: 
![image](https://user-images.githubusercontent.com/97521557/216132276-4edabd1b-6e20-4fd2-a3a0-859cabbba42b.png)